lib/ansible/modules/cloud/openstack/os_network.py

```diff
@@ -198,11 +198,18 @@ def main():
     provider_network_type = module.params['provider_network_type']
     provider_segmentation_id = module.params['provider_segmentation_id']
     project = module.params.get('project')
-    port_security_enabled = module.params.get('port_security_enabled')
-    mtu = module.params.get('mtu')
-    dns_domain = module.params.get('dns_domain')
 
-    sdk, cloud = openstack_cloud_from_module(module)
+    net_create_kwargs = {
+        'port_security_enabled': module.params.get('port_security_enabled'),
+        'mtu_size': module.params.get('mtu')
+    }
+
+    dns_domain = module.params.get('dns_domain')
+    if dns_domain is not None:
```

Comment:
could be just `if module.params['dns_domain'] is not None:`

Reply:
Done.